ooyala_player/public/js/ooyala_player.js

```diff
@@ -183,6 +183,17 @@ function OoyalaPlayerBlock(runtime, element) {
                     time: this.player.getPlayheadTime(),
                     player_state: this.player.getState()
                 });
+                console.log('fullscreenchanged')
+                var playing = this.player.isPlaying();
+                console.log('playing: ' + playing)
+                if (playing == true) {
+                  $('#' + this.data.domId).find('.oo-icon-pause-slick').click()
+                  setTimeout(() => {$('#' + this.data.domId).find('.oo-icon-play-slick').click()}, 0.3*1000)
```

Comment:
did we confirm that arrow function is transpiling to plain javascript?

Reply:
Confirm